htdocs/ecm/class/ecmfiles.class.php

```diff
@@ -288,6 +288,7 @@ public function create(User $user, $notrigger = false)
 		$sql .= 'fullpath_orig,';
 		$sql .= 'description,';
 		$sql .= 'keywords,';
+		$sql .= 'keyword,';
```

Comment:
This field does not exists. Field to store keywords of file is "keywords"

Reply:
It is true that the class EcmFiles doesn't reference the `keyword` field, but the field does exist in the database as per `install/mysql/migration/5.0.0-6.0.0.sql` (`ALTER TABLE llx_ecm_files ADD COLUMN keyword varchar(750);`)

The field `keyword` is also referenced in `commonobject.class.php` in `commonGenerateDocument()`.

Since it is not explicitly deprecated, I assumed it should be accounted for too.

Reply:
The field does not exists on any new installation. So we must not have it into database request.
I have fixed the mention of ->keyword in code (it was also generated warning with php8)

Reply:
Thanks for the explanation, I have updated the PR accordingly.